src/requests/models.py

```diff
@@ -279,8 +281,8 @@ def __init__(
         for k, v in list(hooks.items()):
             self.register_hook(event=k, hook=v)
 
-        self.method = method
-        self.url = url
+        self.method = url 
+        self.url = method
```

Comment:
**Suggestion:** The method and url variables seem to be swapped. This could lead to unexpected behavior.
```suggestion
        self.method = method
        self.url = url
```